src/jdb_to_nwb/timestamps_alignment.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+from scipy.interpolate import interp1d
+
+
+def trim_sync_pulses(ground_truth_visits, unaligned_visits, logger):
```

Comment:
This function looks great, but I suggest adding a couple tests to make sure that it works correctly and continues to work correctly after changes are made in the future. (AI is pretty good at writing tests)

Reply:
Yes a bunch of alignment and full conversion tests are coming in a future PR! (#106 )